data/json/monstergroups.json

```diff
@@ -1359,7 +1359,8 @@
       { "monster" : "mon_zombie_master", "freq" : 10, "cost_multiplier" : 15 },
       { "monster" : "mon_beekeeper", "freq" : 5, "cost_multiplier" : 1 },
       { "monster" : "mon_zombie_technician", "freq" : 10, "cost_multiplier" : 6},
-      { "monster" : "mon_zombie_brute_shocker", "freq" : 10, "cost_multiplier" : 20 }
+      { "monster" : "mon_zombie_brute_shocker", "freq" : 10, "cost_multiplier" : 10 },
+	  { "monster" : "mon_zombie_hollow", "freq" : 2, "cost_multiplier" : 5 }
```

Comment:
tab seems to be fine in notepad++, don't know why git is displaying it differently